### PR TITLE
Fix create from key bundle

### PR DESCRIPTION
**Pull request by nplasterer**
Allow bundles to be created without requiring a signer.

### CODE REVIEW

File: library/src/main/java/org/xmtp/android/library/Client.kt [hunk 1]
```diff
@@ -376,13 +376,15 @@ class Client() {
 
         if (v3Client != null) {
             v3Client.signatureRequest()?.let { signatureRequest ->
-                if (account != null) {
-                    account.sign(signatureRequest.signatureText())?.let {
-                        signatureRequest.addEcdsaSignature(it.rawData)
+                if (!signatureRequest.isReady()) {
```
Comment (neekolas) [inline library/src/main/java/org/xmtp/android/library/Client.kt:379]: Have you tried removing this `isReady` check. I'm still unconvinced it actually matters with the way you're using `libxmtp`

**Review by neekolas — APPROVED** [review]

File: library/src/main/java/org/xmtp/android/library/Client.kt (hunk 1 above)
Comment (nplasterer) [reply library/src/main/java/org/xmtp/android/library/Client.kt]: Yep doesn't seem to be needed removed it.

File: library/src/main/java/org/xmtp/android/library/Client.kt [hunk 2]
```diff
@@ -302,10 +299,10 @@ class Client() {
         account: SigningKey?,
         appContext: Context?,
         privateKeyBundleV1: PrivateKeyBundleV1,
-        legacyIdentitySource: LegacyIdentitySource,
-        accountAddress: String,
+        address: String,
     ): Pair<FfiXmtpClient?, String> {
         var dbPath = ""
+        val accountAddress = address.lowercase()
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/Client.kt:305]: @neekolas I still think there is probably an improvement on the libxmtp side for normalizing addresses maybe in the `getInboxIdForAddress`

Comment (neekolas) [reply library/src/main/java/org/xmtp/android/library/Client.kt:305]: For sure

**Pull request merged**